aws-rotate-keys/orb.yml

```diff
@@ -36,6 +36,29 @@ commands:
       - run:
           name: Rotate AWS keys
           command: |
+            # Check parameters have been provided
+            has_missing_parameters=false
+            missing_param_message="Error: Please provide a non-empty string for parameters:"
+            if [ -z << parameters.aws-username >> ]; then
+              has_missing_parameters=true
+              missing_param_message="$missing_param_message\n- aws-username"
+            fi
+            if [ -z << parameters.circleci-token >> ]; then
+              has_missing_parameters=true
+              missing_param_message="$missing_param_message\n- circleci-token"
+            fi
+            if [ -z << parameters.aws-access-key-id-var >> ]; then
+              has_missing_parameters=true
+              missing_param_message="$missing_param_message\n- aws-access-key-id-var"
+            fi
+            if [ -z << parameters.aws-secret-access-key-var >> ]; then
+              has_missing_parameters=true
+              missing_param_message="$missing_param_message\n- aws-secret-access-key-var"
+            fi
```

Comment:
These two default to env variables, so maybe you could mention the name of the env variables in those error messages, e.g. something like this? `- aws-access-key-id-var or AWS_ACCESS_KEY_ID environment variable` etc.